### ISSUE TITLE
Adding other parameter to query from js

### ISSUE THREAD

**Issue by ichyr** [maxivak/bootstrap3_autocomplete_input#11]
Hi!

I have the two connected inputs. First select filed of study and then, select one of the nobel winners from that field. I use autocomplete to interact with the list of fields of study. I choose one and then I enable second input so the user can search by the name or surname through direction winners.

In order to implement second autocomplete I need to pass from the server some additional data - id of direction selected. Is this possible to do natively?

In other words I need to be able to set from JS side parameter _direction_, so I can work with them in my _results_controller.rb_

```
    params[:q] = params[:q] || ''
    params[:direction] = params[:q] || ''
```

**Comment by maxivak** [comment]
use custom controller method to control what you get from database.

see example - https://github.com/maxivak/bootstrap3_autocomplete_input/wiki/search-custom-method

set variables for your case:

```
options = {
      ...
      order: "name #{params[:direction]}"
  }

...
where = "name like '#{params[:q]}%'"
```
